src/pages/Map.js

```diff
@@ -14,6 +14,7 @@ const containerStyle = {
   top: '50.4px'
 };
 
+//TODO: pull these from firebase
```

Comment:
add someone's name here as an action item for who's responsible. Is this a TODO for you, namrata, me, etc.